### ISSUE TITLE
Additional documents request 

### ISSUE THREAD

**Issue by jurtlest** [knights-lab/SHOGUN#40]
Hi Knights-lab, 

I'm recently interested in analyzing shallow shotgun metagenomics sequencing, and I came across SHOGUN and thought it's interesting to try. 

However, I'm having trouble finding any actual vignettes or pages showing the full pipeline and flow through of SHOGUN process from sequencer fastq to some preliminary analysis and plots?? 

I'm relatively new to shotgun sequencing, and I get that SHOGUN is a one line command. But it will be very helpful to see the code itself and the product. For example, the setup of the required directories are rather arbitrary to novice people like me. And what files are needed etc. 

In short, I have a directory of R1 and R2 fastq files that I'm hoping to analyze, but struggle to know where to begin. If there's any information you can point me to that will be greatly appreciated ! 

Thanks

**Comment by rahel31** [comment]
Hi @Justice-Lu ,
Your question has not been answered, but I was wondering did you manage to make this pipeline to work? If yes, would you mind sharing your experience, as there are no tutorials still available. 
Thank you in advance!
